fbw-common/src/wasm/systems/systems/src/hydraulic/bypass_pin.rs

```diff
@@ -0,0 +1,42 @@
+use super::nose_steering::Pushback;
+use crate::simulation::{
+    InitContext, Read, SimulationElement, SimulatorReader, SimulatorWriter, VariableIdentifier,
+    Write,
+};
+
+pub struct BypassPin {
+    gsx_pin_state_id: VariableIdentifier,
+    gsx_pin_inserted: bool,
+    nw_strg_disc_memo_id: VariableIdentifier,
+
+    bypass_pin_inserted: bool,
```

Comment:
style: we tend to group VariableIdentifiers at the top, and then the rest of the state below.

```suggestion
    gsx_pin_state_id: VariableIdentifier,
    nw_strg_disc_memo_id: VariableIdentifier,

    gsx_pin_inserted: bool,
    bypass_pin_inserted: bool,
```